README.md

```diff
@@ -108,7 +108,6 @@ File|Description
 [yt-dlp_x86.exe](https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp_x86.exe)|Windows (Win7 SP1+) standalone x86 (32-bit) binary
 [yt-dlp_min.exe](https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp_min.exe)|Windows (Win7 SP1+) standalone x64 binary built with `py2exe`<br/> ([Not recommended](#standalone-py2exe-builds-windows))
 [yt-dlp_linux](https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp_linux)|Linux standalone x64 binary
-[yt-dlp_linux.zip](https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp_linux.zip)|Unpackaged Linux executable (no auto-update)
```

Comment:
Why are we dropping it?

Reply:
Mamba build suddenly didnt want to work anymore and so we had to switch to a different strategy (statically linked musl libc builds). I have a plan to potentially bring them back but it will require some research and some trial and error. Bringing back working builds seemed of more immediate importance.

Reply:
I have seen the discussion. But the zip build will be the same workflow with `pyinstaller --one-dir`, no? Sure, fixing it is not a priority, but I thought it'd be trivial

Reply:
Here is a good overview of what staticx does: https://github.com/JonathonReinhart/staticx/issues/239#issuecomment-1601960532

It works similarly to pyinstaller `--onefile`; it's just another layer of bundling on top with some `RPATH`+`INTERP` special sauce. The maintainer also says [here](https://github.com/JonathonReinhart/staticx/issues/183#issuecomment-886337273) that it's not possible to use staticx with pyinstaller `--onedir` builds, so we would need to get very inventive to prove them wrong(?)

Reply:
On that note, I think the static exe is a bit problematic since we are doubling the startup time and disk writes to /tmp. I want to try replacing the `linux_arm` job with a CentOS 7 build flow; if that's successful, and if we can't find a way to optimize the static exe, then maybe we could build the `linux_exe` using the CentOS 7 framework as well.